test/e2e/performanceprofile/functests/0_config/config.go

```diff
@@ -134,6 +135,14 @@ var _ = Describe("[performance][config] Performance configuration", Ordered, fun
 		format.Object(performanceProfile, 2)
 	})
 
+	It("should create SRI-OV devices if they are exist", func() {
```

Comment:
typo: `SR-IOV` (here and elsewhere): https://en.wikipedia.org/wiki/Single-root_input/output_virtualization

Reply:
I'll keep forgetting :+1: